compatibility_server/deployment/deploy.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "3"
+  creationTimestamp: 2018-06-15T20:08:48Z
+  generation: 8
+  labels:
+    run: compatibility-server
+  name: compatibility-server
+  namespace: default
+  resourceVersion: "19908269"
```

Comment:
Where do these values from? For example, "creationTimestamp", "resourceVersion", "uid".